site/cypress/e2e/Cypress-Feature/viewing_version_conflict.spec.js

```diff
@@ -0,0 +1,58 @@
+import { buildUrl } from '/cypress/support/utils.js';
+
+const scores = [];
+describe('Test revolving around the student side and whether or not they should see a version conflict', () => {
+    it('Student should have a version conflict', () => {
+        cy.login('student');
+        cy.visit(['sample', 'gradeable', 'grades_released_homework_onlytaEC']);
+
+        cy.get('[data-testid="version-conflict-version-box"]').contains('Note: The version you have selected to be graded is not the version graded by the instructor/TAs.');
```

Comment:
It's better to use assertions in this situation, cy.get(...).should('contain', '...'), same with the rest of this PR.